### PR TITLE
Add related codes to local codes

### PR DESCRIPTION
**Pull request by Remg**

### CODE REVIEW

File: tests/Referent/ManagedAreaUtilsTest.php [hunk 1]
```diff
@@ -2,10 +2,14 @@
 
 namespace Tests\AppBundle\Referent;
 
+use AppBundle\Entity\Adherent;
 use AppBundle\Entity\Committee;
 use AppBundle\Referent\ManagedAreaUtils;
 use PHPUnit\Framework\TestCase;
 
+/**
+ * @group huehue
```
Comment (illusionOfParadise) [inline tests/Referent/ManagedAreaUtilsTest.php]: `huehue` ? 😄

Comment (thomasbisignani) [reply tests/Referent/ManagedAreaUtilsTest.php]: :joy: :medal_sports:

File: src/Utils/AreaUtils.php [hunk 2]
```diff
@@ -44,8 +50,33 @@ public static function getCodeFromCountry(string $country): string
...
+        if (self::isCorsicaCode($code)) {
+            $relatedCodes[] = self::PREFIX_POSTALCODE_CORSICA;
+        }
+
+        return $relatedCodes;
+    }
+
+    private static function isParisCode(string $code): bool
+    {
+        return self::PREFIX_POSTALCODE_PARIS_DISTRICTS === substr($code, 0, 2);
```
Comment (B-Galati) [inline src/Utils/AreaUtils.php]: `mb_substr` perhaps ?

File: src/Utils/AreaUtils.php [hunk 3]
```diff
@@ -44,8 +50,33 @@ public static function getCodeFromCountry(string $country): string
...
+    }
+
+    private static function isCorsicaCode(string $code): bool
+    {
+        return in_array($code, [self::CODE_CORSICA_A, self::CODE_CORSICA_B]);
```
Comment (B-Galati) [inline src/Utils/AreaUtils.php]: Adding `true` to the third argument of `in_array` would be more accurate.

File: tests/Referent/ManagedAreaUtilsTest.php (hunk 1 above)
Comment (B-Galati) [reply tests/Referent/ManagedAreaUtilsTest.php]: `phpunit --filter ManagedAreaUtilsTest` otherwise :-)

Comment (Remg) [reply tests/Referent/ManagedAreaUtilsTest.php]: good catch
:trollface:

File: tests/Utils/AreaUtilsTest.php [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+     */
+    public function testGetCodeFromCountry($country, $expectedCode): void
+    {
+        $this->assertEquals($expectedCode, AreaUtils::getCodeFromCountry($country));
+    }
+
+    public function provideCountries(): \Generator
+    {
+        yield ['DE', 'DE'];
+        yield ['IT', 'IT'];
```
Comment (thomasbisignani) [inline tests/Utils/AreaUtilsTest.php]: :1st_place_medal: :spaghetti:

**Pull request merged**